uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/tasks-list.spec.js

```diff
@@ -82,6 +83,40 @@ define([
           });
         });
 
+        describe('filterByDateField()', function () {
+          var filteredTaskList;
+
+          describe('filtering tasks in between 2017-04-10 and 2017-04-20', function () {
```

Comment:
I am not quite understand why `2017-04-10 - 2017-04-20` returns 3 documents and `2017-04-23 and 2017-04-25` returns 0. May be it would be better to describe tests like that:

describe: 'filtering tasks for the time period before due date'
it: 'returns filtered tasks by due date'

describe: 'filtering tasks for the time period after due date'
it: 'does not return filtered tasks'

Reply:
Will update it!